src/vivarium_conic_lsff/components/fortification/parameters.py

```diff
@@ -105,3 +105,99 @@ def sample_folic_acid_relative_risk(location: str, draw: int) -> float:
     return sample_lognormal_distribution(seed, RELATIVE_RISK)
 
 
+VITAMIN_A_COVERAGE = {
+    # TODO - numbers for Ethiopia are stubbed
+    'Ethiopia': [
+        {
+            'baseline': BetaParams(
+                upper_bound=1,
+                lower_bound=0,
+                alpha=0.1,
+                beta=9.9,
+            ),
+            'intervention_start': BetaParams(
+                upper_bound=1,
+                lower_bound=0,
+                alpha=0.5,
+                beta=3.1,
+            ),
+            'intervention_end': BetaParams(
+                upper_bound=1,
+                lower_bound=0,
+                alpha=0.8,
+                beta=2.36,
+            ),
+            'weight': 1,
+        },
+    ],
+    'India': [
+        {
+            'baseline': BetaParams.from_statistics(
+                mean=0.243,
+                upper_bound=0.279,
+                lower_bound=0.211,
+            ),
+            'intervention_start': BetaParams.from_statistics(
+                mean=0.894,
+                upper_bound=0.918,
+                lower_bound=0.87,
```

Comment:
NIT: Use the same number of significant digits for all the data params

Reply:
Done.